docs/getting-started/os-support.rst

```diff
@@ -28,13 +28,13 @@ ScyllaDB Open Source
 +----------------------------+-------------+---------------+---------+---------------+--------------+--------------+
 | Linux Distributions        |Ubuntu       | Debian        | CentOS /| Rocky /       | Oracle Linux | Amazon Linux |
 |                            |             |               | RHEL    | RHEL          |              |              |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   | 7       |   8   |   9   |  7.6  | 8.1  | 2            |
-+============================+======+======+=======+=======+=========+=======+=======+=======+======+==============+
-|   5.4                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |v|   | |v|   | |v|  | |v|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
-|   5.2                      | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |x|   | |x|   | |x|  | |x|          |
-+----------------------------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
++----------------------------+------+------+-------+-------+---------+-------+-------+--------------+--------------+
+| ScyllaDB Version / Version |20.04 |22.04 |  10   |  11   |   7     |   8   |   9   | 8.1          | 2            |
```

Comment:
I find it strange we have OEL 8.1 here, which is quite ancient (8.8! is the latest - https://docs.oracle.com/en-us/iaas/images/oracle-linux-8x/ )

Reply:
I don't remember that we removed CentOS 7 support from 5.4.

Reply:
Sent you the reference for this decision (in a separate issue)